### DISCUSSION TITLE
Will we see support for this build in Visual Studio Code any time soon?

### DISCUSSION THREAD

**Discussion by NathanHannon** [microsoft/openjdk#22] [Q&A] [answered]
Do we have any idea of when we will see support for this build in Visual Studio Code, or is that up to Red Hat, since they make the Java extension?

**Comment by brunoborges** [comment, marked as answer]
Hi @NathanHannon,

The Red Hat extension for Java Language Support already works on top of the Microsoft Build of OpenJDK. Configuration can be done manually. 

But the extension that helps configure the overall Java settings is maintained by Microsoft, and we will surface our build of OpenJDK as an option after we have GA.

Thanks for raising interest in that.

I've created the following ticket to track this: https://github.com/microsoft/vscode-java-pack/issues/594

// @akaroml @yaojinyang

**Reply by NathanHannon** [reply]
Thanks for the quick reply! Looking forward to seeing it.
